src/System.Net.WebSockets.WebSocketProtocol/src/System.Net.WebSockets.WebSocketProtocol.csproj

```diff
@@ -0,0 +1,22 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>netstandard2.0;net48</TargetFrameworks>
```

Comment:
@ericstj Am I targeting the wrong .NET Framework version? I can't remember why I chose net48, and I don't see any comments in the PR asking to use net48.

I see that in 5.0, we were targeting net461: https://github.com/dotnet/runtime/blob/4aadfea70082ae23e6c54a449268341e9429434e/src/libraries/System.Net.WebSockets.WebSocketProtocol/src/System.Net.WebSockets.WebSocketProtocol.csproj#L4

So I suspect I should instead target net462 like we've been doing all along.

Reply:
Probably `$(NetFrameworkMinimum)`

Reply:
Yep, it works with NetFrameworkMinimum, seems to be the correct choice. I also added NetMinimum and kept netstandard2.0.

I remember now why I had net48: Because the tests use APIs that are not available in net462, only in net48. But the tests can continue using net48, only the src will use net462.